frontend/src/metabase-types/api/mocks/dashboard.ts

```diff
@@ -26,6 +26,7 @@ export const createMockDashboard = (opts?: Partial<Dashboard>): Dashboard => ({
   archived: false,
   public_uuid: null,
   enable_embedding: false,
+  tabs: [],
```

Comment:
It seems that the BE always returns this to us, and the test would fail because the FE code checks the `tabs.length > 0`